src/zodbpickle/tests/pickletester_3.py

```diff
@@ -779,14 +778,14 @@ def test_structseq(self):
                 u = self.loads(s)
                 self.assertEqual(t, u)
 
-    @unittest.skipUnless(_PY33, "only for Python >= 3.3")
+    @unittest.skipUnless(_PY34, "only for Python >= 3.4")
```

Comment:
Perhaps these decorators can be simplified. We don't care about the specific point release of Python 3, at this point we just care that we're running on Python 3 period. So they could all be something like `@skipUnless(PY3, "Only for Python 3")`. 

Personally in those cases where the condition and message is constant I like to create a helper decorator (untested):
```python
def skipOnPython2(f):
    if PY3:
     return f
   return unittest.skip("Requires Python 3")(f)

@skipOnPython2
def test_notimplemented():
  ...
```

I could imagine avoiding these changes if this were a test file we were trying to keep in sync with the stdlib, but I'm not sure that's true.

Reply:
Actually, AFAICS, this code is only used on Python 3 (`test_pickle.py` either imports `pickletester_2` or `pickletester_3`, depending on system, but not both.) So we should be able to remove those decorators entirely now.

Reply:
@jamadden You are right, thank you for catching this.